loadtest/store/memstore/store.go

```diff
@@ -150,7 +150,7 @@ func (s *MemStore) Channels(teamId string) ([]model.Channel, error) {
 
 func (s *MemStore) SetChannels(channels []*model.Channel) error {
 	if channels == nil {
-		return errors.New("channels shouldn't be nil")
+		return errors.New("memstore: channels shouldn't be nil")
```

Comment:
unrelated: I just noticed that elsewhere we use "should not", but here we use "shouldn't". Let's change this to "should not" while we are here. Or feel free to do it in a separate PR too.